src/sentry/static/sentry/app/utils/statics-setup.tsx

```diff
@@ -4,11 +4,13 @@
 declare var __webpack_public_path__: string;
 
 /**
- * Set the webpack public path at runtime. The __sentryGlobalStaticPrefix will
- * be declared in layout.html.
+ * Set the webpack public path at runtime. This is necessary so that imports can be resolved properly
```

Comment:
Tiny nit but can you `gq` this to wrap like the rest of the text

Reply:
I can do this in a follow-up